static-analysis/autodispose-lint/src/main/kotlin/com/uber/autodispose/lint/AutoDisposeDetector.kt

```diff
@@ -68,17 +68,7 @@ class AutoDisposeDetector : Detector(), SourceCodeScanner {
         Category.CORRECTNESS,
         10,
         Severity.ERROR,
-        // We use the overloaded constructor that takes a varargs of `Scope` as the last param.
-        // This is to enable on-the-fly IDE checks. We are telling lint to run on both
-        // JAVA and TEST_SOURCES in the `scope` parameter but by providing the `analysisScopes`
-        // params, we're indicating that this check can run on either JAVA or TEST_SOURCES and
-        // doesn't require both of them together.
-        // From discussion on lint-dev https://groups.google.com/d/msg/lint-dev/ULQMzW1ZlP0/1dG4Vj3-AQAJ
-        // TODO: Remove after AGP 3.4 release when this behavior will no longer be required.
-        Implementation(AutoDisposeDetector::class.java,
-            EnumSet.of(Scope.JAVA_FILE, Scope.TEST_SOURCES),
-            EnumSet.of(Scope.JAVA_FILE),
-            EnumSet.of(Scope.TEST_SOURCES))
+        Implementation(AutoDisposeDetector::class.java, EnumSet.of(Scope.JAVA_FILE, Scope.TEST_SOURCES))
```

Comment:
Do the IDE checks still work?

Reply:
They don't seem to work at all when I try them in `CatchUp`, even running the lint task from command line 🤔